assembly/src/parsers/u32_ops.rs

```diff
@@ -25,7 +25,7 @@ pub fn parse_u32test(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), As
 /// Translates u32testw assembly instruction to VM operations.
 ///
 /// Implemented by executing `DUP U32SPLIT SWAP DROP EQZ` on each element in the word,
-/// with a total of 28 VM cycles.
+/// with a total of 23 VM cycles.
```

Comment:
I'd prefer to add just a bit more detail to the whole comment. Maybe something like:

"Implemented by executing `DUP U32SPLIT SWAP DROP EQZ` on each element in the word and combining the results using `AND` operation (total of 23 VM cycles).